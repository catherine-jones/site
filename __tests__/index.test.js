/* eslint-env jest */

import React from 'react'
import { shallow } from 'enzyme';

import Home from '../pages/index.js'

describe('Homepage', () => {
  it('Shows a positioning statement', () => {
    expect(shallow(<Home />).contains("Lightning fast testing and delivery")).toBe(true);
  })
})