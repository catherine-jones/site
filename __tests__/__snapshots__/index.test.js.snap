// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "oh hai world" 1`] = `
<h1
  className="sc-bdVaJa iLyilk"
>
  oh hai world
</h1>
`;
